api/app/src/v2/Registration/Assembler/SiriusToLayDeputyshipDtoAssembler.php

```diff
@@ -46,7 +47,14 @@ private function buildDto(array $data): LayDeputyshipDto
         return
             (new LayDeputyshipDto())
                 ->setCaseNumber($data['Case'])
+                ->setClientFirstname($data['ClientFirstname'] ?: '')
```

Comment:
I think if we're setting ClientFirstname to required data above, then we probably shouldn't be allowing it to be set to a blank string here and should simply set the value to `$data['ClientFirstname']`

Reply:
Nice!